corehq/apps/user_importer/importer.py

```diff
@@ -771,15 +755,20 @@ def remove_invited_web_user(domain, username):
     invitation.delete()
 
 
-def remove_web_user_from_domain(domain, user, username, upload_user, is_web_upload=False):
+def remove_web_user_from_domain(domain, user, username, upload_user, user_change_logger=None,
+                                is_web_upload=False):
     if not user or not user.is_member_of(domain):
         if is_web_upload:
             remove_invited_web_user(domain, username)
+            if user_change_logger:
+                user_change_logger.add_change_message(_("Invitation revoked"))
         else:
             raise UserUploadError(_("You cannot remove a web user that is not a member of this project."
                                     " {web_user} is not a member.").format(web_user=user))
     elif username == upload_user.username:
-        raise UserUploadError(_("You cannot remove a yourself from a domain via bulk upload"))
+        raise UserUploadError(_("You cannot remove yourself from a domain via bulk upload"))
     else:
         user.delete_domain_membership(domain)
         user.save()
+        if user_change_logger:
+            user_change_logger.add_change_message(_("Removed from domain"))
```

Comment:
```suggestion
            user_change_logger.add_change_message(_("Removed from domain '{domain}'").format(domain=domain))
```

Reply:
Thanks for pointing that out. 
Updated this and other places that could use the domain name
https://github.com/dimagi/commcare-hq/pull/29935/commits/7be66111d94d17a023e2161bb84ec416d876e7c0